src/main/java/org/jabref/gui/util/component/TagBar.java

```diff
@@ -27,18 +27,21 @@
 
     private final ListProperty<T> tags;
     private StringConverter<T> stringConverter;
-    @FXML private TextField inputTextField;
-    @FXML private HBox tagList;
+    @FXML
+    private TextField inputTextField;
+    @FXML
+    private HBox tagList;
     private BiConsumer<T, MouseEvent> onTagClicked;
+    private boolean allowsMultiple = true;
```

Comment:
Maybe, the enum could also be used here.

Reply:
sorry... but it seems like if we delete this boolean we will have to give Tagbar the access to the field from LinkedEntriesEditor instead... is it better ?

Reply:
If you declare as `  Set<FieldProperty> fieldProperties`;  you can simply pass field.getProperties as parameter 
You would then simply move the check  contains down the to the TagBar.